src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1213,35 +1213,29 @@
     <value>Symmetric encryption should always use a non-repeatable initialization vector to prevent dictionary attacks.</value>
   </data>
   <data name="DataTableReadXmlMessage" xml:space="preserve">
-    <value>The method '{0}' is insecure when deserializing untrusted data. If deserializing untrusted data, replace with TBD.</value>
+    <value>The method '{0}' is insecure when deserializing untrusted data.</value>
   </data>
   <data name="DataTableReadXmlTitle" xml:space="preserve">
-    <value>Do not use insecure deserialization with DataTable.ReadXml()</value>
+    <value>Do not use insecure deserialization with DataTable.ReadXml() with untrusted data</value>
```

Comment:
```suggestion
    <value>Do not use insecure deserialization with 'DataTable.ReadXml()' with untrusted data</value>
```

Reply:
I am not a native English speaker but the sentence reads weird to me because of the two consecutive `with`.